jenkins/opensearch-dashboards/integ-test.jenkinsfile

```diff
@@ -256,14 +256,13 @@ pipeline {
                                                         localPath: "${WORKSPACE}/${distribution}",
                                                         switchUserNonRoot: "${switch_user_non_root}"
                                                     )
-                                                    String closeCommentMessage = "Closing the issue as the Integration Test passed for ${local_component}<br>Version: ${version}<br>Distribution: ${distribution}<br>Architecture: ${architecture}<br>Platform: ${platform}<br><br>Please check the logs: ${RUN_DISPLAY_URL}<br><br> *"
                                                     if (env.updateGithubIssues.toBoolean()) {
-                                                        closeGithubIssue(
+                                                        updateGitHubIssueLabels(
                                                             repoUrl: buildManifestObj.getRepo("${local_component}"),
-                                                            issueTitle: "[AUTOCUT] Integration Test failed for ${local_component}: ${version} ${distribution} distribution",
-                                                            closeComment: closeCommentMessage,
-                                                            label: "autocut,v${version},integ-test-failure"
-                                                        )
+                                                            issueTitle: "[AUTOCUT] Integration Test failed for ${local_component}: ${version}",
+                                                            label: "${platform}:${distribution}:${architecture}",
```

Comment:
LGTM, but we need to observe `"${platform}:${distribution}:${architecture}",` how does this look and work with search API's in GitHub, we can also always come back change.

Reply:
I see! Sounds good. Let me know if there is someplace I need to keep watch.